### PR TITLE
add some log messages to inform user when generating the tls certificates

### PR DESCRIPTION
**Pull request by cpanato**
**What type of PR is this?**

/kind cleanup
/kind feature

**Any specific area of the project related to this PR?**
/area cli

**What this PR does / why we need it**:

- add some log messages to inform user when generating the tls certificates
when using this command I felt lost in what was going on and did not see any outputs when the command succeed.

This PR add some logs just to inform the user what is going on

```
$ ./falcoctl tls install
INFO: Generating certificates in /Users/cpanato/code/src/github.com/falcosecurity/falcoctl directory
INFO: Saving server.key to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Saving client.key to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Saving ca.key to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Saving ca.crt to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Saving server.crt to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Saving client.crt to /Users/cpanato/code/src/github.com/falcosecurity/falcoctl
INFO: Done generating the TLS certificates
```

/assign @maxgio92 @leogr 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/tls/install/install.go [hunk 1]
```diff
@@ -32,6 +35,7 @@ const (
 // NewTLSInstallCmd returns the tls install command.
 func NewTLSInstallCmd() *cobra.Command {
 	options := tls.Options{}
+	options.Printer = output.NewPrinter("tls", true, false, os.Stdout)
```
Comment (alacuku) [inline cmd/tls/install/install.go:38]: I suggest not initializing the printer inside the `tls` command, but passing it through the `common.Options` as we do for the other subcommands: https://github.com/falcosecurity/falcoctl/blob/7227ec4afdafeec4d316e13e0bdc95c904604923/cmd/root.go#L58-L61

This way we ensure that the printer is configured using the global flags such as `verbosity & styling`. In order to configure the printer you can still use the https://github.com/falcosecurity/falcoctl/blob/7227ec4afdafeec4d316e13e0bdc95c904604923/pkg/options/common.go#L80 function.

Comment (cpanato) [reply cmd/tls/install/install.go:38]: @alacuku i see, thank you for the feedback! will make the changes

**Comment by cpanato** [comment]
@alacuku @leogr do we care for the `depguard` golangci-lint rule? in other projects we have been disabling that because it is very expensive to maintain, asking that because that check is failing for this PR

**Comment by alacuku** [comment]
> @alacuku @leogr do we care for the `depguard` golangci-lint rule? in other projects we have been disabling that because it is very expensive to maintain, asking that because that check is failing for this PR

Feel free to disable it. It's there because I bumped the `golangci`-lint to the latest version.

**Comment by maxgio92** [comment]
Thanks @cpanato

/lgtm

**Comment by leogr** [comment]
> > @alacuku @leogr do we care for the `depguard` golangci-lint rule? in other projects we have been disabling that because it is very expensive to maintain, asking that because that check is failing for this PR
> 
> Feel free to disable it. It's there because I bumped the `golangci`-lint to the latest version.

Agree

**Review by leogr — APPROVED** [review]

**Pull request merged**